builtin/logical/pki/path_issue_sign.go

```diff
@@ -220,6 +220,7 @@ func (b *backend) pathIssueSignCert(
 
 		if !useCSR {
 			resp.Data["private_key"] = base64.StdEncoding.EncodeToString(parsedBundle.PrivateKeyBytes)
+			resp.Data["private_key_type"] = parsedBundle.PrivateKeyType
```

Comment:
I had spent some time wondering this myself.  The two values would appear to be functionally equivalent.  `*ParsedCertBundle.ToCertBundle()` will copy the value over verbatim.  After reading the documentation, I was still unsure whether there was any semantic difference between these two fields.

My original commit went with the `PrivateKeyType` field from the `ParsedCertBundle` type because, to me, it made the most sense to source this field from the type that was ostensibly created to store DER-encoded data.  The `"der"` expression switch case does not read from `cb` anywhere else, which seemed to fit with this approach.

OTOH, I see now that the `root/generate/...` handler [uses](https://github.com/hashicorp/vault/blob/ff7a1a810ba0dd1e436770e2c345d67d72b4c2c1/builtin/logical/pki/path_root.go#L131) the `PrivateKeyType` field from the `CertBundle` type so I would suppose my understanding is incomplete.  Given the existing pattern and your comment, I have changed this to `cb`.